orbslam3:
  ros__parameters:
    world_frame: "odom"
    odom_frame: "camera_link"
    camera_frame: "camera_color_frame"
    camera_optical_frame: "camera_color_optical_frame"